Calculator.java

```diff
@@ -0,0 +1,88 @@
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public int calculate(String s) {
+        // comment for new commit...
+
+        ArrayList<String> factorials = new ArrayList<String>();
+        ArrayList<String> finalString = new ArrayList<String>();

```

Comment:
Try to do away without ArrayList and any other advanced data structures so far. For learning purposes, we are using arrays-only now